content/geometry/introduction.adoc

```diff
@@ -9,18 +9,18 @@ endif::[]
 
 == Motivation
 
-A 3D asset is a digital entity that includes a 3D geometric representation of an object along with its attributes and metadata. 
+A 3D asset (or **3D Data File**) is a digital entity that includes a 3D geometric representation of an object along with its attributes and metadata. 
 Common formats for storing the geometry and visual materials of 3D models include glTF, FBX, and USD.
 
 However, these formats do not provide a standardized way to structure the geometry or provide extensive metadata storage and exchange capabilities. 
 The ASAM OpenMATERIAL standard addresses these needs.
 
 This chapter includes all geometry related definitions of ASAM OpenMATERIAL, marked in red in <<fig-openmaterial-geometry-overview>>.
 
-The ASAM OpenMATERIAL asset file (for example, "omg_my-model.xoma") contains essential information relevant to the display and interactivity attributes of the 3D model in various computer applications.
+The ASAM OpenMATERIAL **Asset File** (for example, "omg_my-model.xoma") contains essential information relevant to the display and interactivity attributes of the 3D model in various computer applications.
```

Comment:
We could point to the example asset in the examples folder (example_asset.xoma). But this does not have the omg_ prefix, yet. This might need to be added first.

Reply:
Since the prefix isn't there, yet, I'd skip this for now and decide on establishment of links for all relevant artifacts in the next meeting.